index.js

```diff
@@ -1,21 +1,51 @@
 "use strict";
 const request = require("request");
 const log = require("npmlog");
+const fs = require('fs');
+
 const util = require("./app/util");
 const urlGenerator = require("./app/urlGenerator");
+
 const args = process.argv.slice(2);
+const tokenLocation = 'slack.token';
 
 function slackMessage(){
-  if(args.length < 3) {
-    throw new Error("All properties are required 'slack-message <token> <channel> <message>'");
-  } else if (args.length > 3) {
-    throw new Error("No additional properties allowed 'slack-message <token> <channel> <message>'");
+  if (args.length == 1 && args[0] == "delete-token") {
+    deleteToken();
+    return log.info("Token deleted");
+  } else if(args.length == 2 && args[0] == "save-token") {
+    saveToken(args[1]);
+    return log.info("Token saved: " + args[1]);
   }
-  const message = {
-    token: args[0],
-    channel: args[1],
-    text: args[2]
+
```

Comment:
Could you just refactor your token stuff and add it to another file in `app` folder (something like generate token/message dunno).